lib/metered_raven_error_handler.rb

```diff
@@ -16,11 +23,17 @@ def call(ex, original_context)
       # Is this a retryable job?
       is_retryable = context[:retry] || (context[:job] && context[:job][:retry])
 
-      # We notify if this job has been retried at least once.
+      # We notify to Sentry if:
+      # This job has been retried at least once
+      #             OR
+      # Is not set being retried
+      #             OR
+      # Is not one of the errors that should be ignored on first occurrence
+      #
       # Someday we plan to make this number configurable.
       # If this isn't a retryable job, we notify even if this is the first attempt, because there will not be more attempts.
-      if retry_count > 0 || !is_retryable
-        ::Rails.logger.debug("MeteredRavenErrorHandler -- Current retry count: #{retry_count}. Notifying upstream...")
+      if retry_count > 0 || !is_retryable || !errors_to_ignore_on_first_occurrence.include?(ex.class)
```

Comment:
Now that we're only doing the ignore for specific cases, I wonder if this would be more intuitively readable as:
```
unless errors_to_ignore_on_first_occurrence.include?(ex.class) && is_retryable && retry_count < 1
```

(And then the comment becomes: We notify unless it's going to be retried and hasn't been retried already and it's the kind of error we ignore the first time, or something)

Reply:
Hmm I'm not sure, I understand the negations can be a bit confusing but I find myself the `unless` more obscure

Reply:
Yeah, I guess my feeling is: there's a specific situation in which we're skipping the error, but the current way it's written feels more like there are a variety of situations in which we're not skipping :shrug: